visualizer/src/Controls/Tracking/Division.js

```diff
@@ -0,0 +1,251 @@
+  const theme = useTheme();
+  const strokeColor = theme.palette.secondary.main;
+
+  const relations = daughters.map((label) => ({
+    targetId: `daughter${label}`,
+    targetAnchor: 'left',
+    sourceAnchor: 'right',
+    style: { strokeColor, strokeWidth: 1, noCurves: true },
+  }));
+  relations.push({
+    targetId: 'addDaughter',
+    targetAnchor: 'left',
+    sourceAnchor: 'right',
+    style: { strokeColor, strokeWidth: 1, noCurves: true },
+  });
+
+  const select = useSelect();
+  const image = useImage();
+
+  const onClick = (e) => {
+    select.send({ type: 'SET_FOREGROUND', foreground: label });
+    image.send({
+      type: 'LOAD_FRAME',
+      frame: divisionFrame ? divisionFrame - 1 : frames[frames.length - 1],
+    });
+  };
+
+  return (
+    <ArcherElement id='parent' relations={relations}>
+      <Cell label={label} onClick={onClick} />
+    </ArcherElement>
+  );
+}
+
+function Daughter({ label, daughter, divisionFrame }) {
+  const styles = useStyles();
+
+  const select = useSelect();
+  const image = useImage();
+
+  const onClick = () => {
+    select.send({ type: 'SET_FOREGROUND', foreground: daughter });
+    image.send({ type: 'LOAD_FRAME', frame: divisionFrame });
+  };
+
+  return (
+    <Box className={styles.daughter}>
+      <ArcherElement id={`daughter${daughter}`}>
+        <Cell label={daughter} onClick={onClick} />
+      </ArcherElement>
+      <DaughterMenu parent={label} daughter={daughter} />
+    </Box>
+  );
+}
+
+function AddDaughter({ label }) {
+  const styles = useStyles();
+
+  const tracking = useTracking();
+
+  const [open, toggle] = useReducer((v) => !v, false);
+  const anchorRef = useRef(null);
+
+  const handleAddDaughter = () => {
+    tracking.send({ type: 'ADD_DAUGHTER', parent: label });
+    toggle();
+  };
+  const handleNewCell = () => {
+    tracking.send({ type: 'CREATE_NEW_CELL', label });
+    toggle();
+  };
+
+  return (
+    <Box style={{ position: 'relative' }}>
+      {/* point arrow to hidden Avatar so arrows look aligned */}
+      <ArcherElement id='addDaughter'>
+        <Avatar className={styles.cell} style={{ visibility: 'hidden' }} />
+      </ArcherElement>
+      <IconButton className={styles.addButton} onClick={toggle} ref={anchorRef}>
+        <AddCircleOutlineIcon />
+      </IconButton>
+      <Popper open={open} anchorEl={anchorRef.current} placement='bottom-end'>
+        <Paper>
+          <ClickAwayListener onClickAway={toggle}>
+            <MenuList id='add-daughter-menu'>
+              <MenuItem onClick={handleAddDaughter}>Add Daughter</MenuItem>
+              <MenuItem onClick={handleNewCell}>Create New Cell</MenuItem>
+            </MenuList>
+          </ClickAwayListener>
+        </Paper>
+      </Popper>
+    </Box>
+  );
+}
+
+function DaughterMenu({ parent, daughter }) {
+  const tracking = useTracking();
+  const { send } = tracking;
+
+  const [open, toggle] = useReducer((v) => !v, false);
```

Comment:
Minor: IDK if this naming is standard, but it might be useful to change the `toggle` to `toggleOpen` (or whatever we are toggling). It wasn't clear to me at first.